.github/workflows/cli.yaml

```diff
@@ -55,4 +57,4 @@ jobs:
           cluster-resources: nodes,namespaces,
           namespace-resources: configmaps,pods,svc
           artifact-name: logs-${{ matrix.k8s}}
-
+          
```

Comment:
```suggestion

```
@Akash-Singh04 
Refer to https://gist.github.com/camh-/1bebfcff1b0f814e9b191edc60d5206b, a github text file should end with a new line. The current `cli.yaml` file is actually end with two new lines, so you can choose to delete just one of the lines, or leave them alone. What you've done is actually remove a new line and add a line that contains invisible characters

Reply:
done.